src/functions/indGraphSkinny.jl

```diff
@@ -0,0 +1,54 @@
+# This implements the variant when A is dense and m > n
+
+struct IndGraphSkinny{T <: RealOrComplex} <: IndGraph
+  m::Int
+  n::Int
+  A::AbstractArray{T,2}
+  AA::AbstractArray{T,2}
+  F::Base.LinAlg.Cholesky #LDL factorization
+end
+
+function IndGraphSkinny(A::AbstractArray{T,2}) where {T <: RealOrComplex}
+  m, n = size(A)
+  AA = A' * A
+
+  F = LinAlg.cholfact(AA + speye(n))
```

Comment:
Sparse eye seems weird, I assume julia will convert it to dense (since AA is dense) and then add them. We might want to avoid allocating so much memory here, but it might not be important. The constructors are not assumed to be called to often.